src/pathwayvis/components/map/map.component.ts

```diff
@@ -413,16 +413,28 @@ class MapComponentCtrl {
     if (selected.method.id === 'fva' || selected.method.id === 'pfba-fva') {
 
       // const fvaData = reactionData;
-      const fvaData = _.pickBy(reactionData, (d) => Math.abs((d.upper_bound + d.lower_bound) / 2) > 1e-7);
+      const fvaData = reactionData;
+      for (const i in fvaData) {
+        if (Math.abs((fvaData[i].upper_bound + fvaData[i].lower_bound) / 2) < 1e-7) {
+          delete fvaData[i];
+        }
+      }
+      // const fvaData = _.pickBy(reactionData, (d) => Math.abs((d.upper_bound + d.lower_bound) / 2) > 1e-7);
```

Comment:
please don't leave commented code, and delete the one above. https://github.com/DD-DeCaF/metabolica-ui-map/pull/138/files#diff-62c06d0a48047fe88322f20be6e856e2R415